src/main/java/com/nike/cerberus/command/core/RestoreCerberusBackupCommand.java

```diff
@@ -22,7 +22,7 @@
 import com.nike.cerberus.operation.Operation;
 import com.nike.cerberus.operation.core.RestoreCerberusBackupOperation;
 
-import static com.nike.cerberus.command.core.WhitelistCidrForVpcAccessCommand.COMMAND_NAME;
+import static com.nike.cerberus.command.core.RestoreCerberusBackupCommand.COMMAND_NAME;
```

Comment:
Are we importing static from the same class?

Reply:
@tlisonbee yes, it's because the `@Parameters` annotation sits outside of the class declaration